lib/ansible/module_utils/facts/virtual/linux.py

```diff
@@ -201,7 +201,7 @@ def get_virtual_facts(self):
                 virtual_facts['virtualization_type'] = 'virtualbox'
                 found_virt = True
 
-        if bios_vendor in ('Amazon EC2', 'DigitalOcean', 'Hetzner'):
+        if bios_vendor in ('Amazon EC2', 'DigitalOcean', 'Hetzner', 'Linode'):
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.